src/listener.mjs

```diff
@@ -57,17 +37,35 @@ async function runListener() {
   const dbUpdates = new DbUpdates(nodeDb);
   await subscribeToGatewayNotifications(dbUpdates);
 
-  app.use(bodyParser());
-  app.use(router.routes());
-  app.use(router.allowedMethods());
-
-  app.context.nodeDb = nodeDb;
+  const app = createApp(nodeDb);
 
   app.listen(port);
-
   logger.info(`Listening on port ${port}`);
 }
 
+async function onNewState(message, dbUpdates) {
```

Comment:
either refactor this to a separate file (like the onInteraction.mjs), or move the content of the onInteraction.mjs here.

Reply:
done to separate file!